### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by MooSayed1**

### CODE REVIEW

File: cod/graph/dfs.cpp
Comment (AnonMiraj) [inline cod/graph/dfs.cpp:1]: Will we ever need manual dfs ?
I dont think it has any advantages over normal recursion.
and dfs is so simple in my opinion it is not necessary to include it in the reference

File: cod/graph/dijkstra.cpp
Comment (AnonMiraj) [inline cod/graph/dijkstra.cpp:1]: Remove unnecessary include and define.

File: reference_material.pdf
Comment (AnonMiraj) [inline reference_material.pdf:1]: dont upload the resulted pdf 
i will use a github action to auto update it when a merge happen

**Review by AnonMiraj — CHANGES_REQUESTED** [review]

File: cod/graph/dfs.cpp
Comment (MooSayed1) [reply cod/graph/dfs.cpp:1]: i want it in one problem there not a problem if we add it

**Review by AnonMiraj — APPROVED** [review]

**Pull request merged**